### DISCUSSION TITLE
JWE Encypt support string type in plainText parameter

### DISCUSSION THREAD

**Discussion by adrianlivr** [panva/jose#156] [Q&A] [answered]
I need send a request with JWE using plainText as json, but actually the plainText parameter support only Uint8Array.
I want know if is posible define plainText as "Uint8Array | string"?

**Comment by panva** [comment, marked as answer]
Did you open the docs? It shows how in each jwe example. Also, Buffer instances are Uint8Arrays in Node.

All you have to do in node is pass `Buffer.from(yourString)`, alternatively, if your string uses encoding other than utf-8, pass `Buffer.from(yourString, encoding)`.

The API intentionally only accepts and returns UInt8Array so that it does not have to deal with encoding.

**Comment by adrianlivr** [comment]
![Screenshot_20210121_180232](https://user-images.githubusercontent.com/60880274/105412485-980c2980-5c13-11eb-8968-62458a1cc63b.png)

It's work
Thanks!

I read the docs, but I'm not an expert in binary data or buffers
